tools/XmlDocToMarkdown/XmlDocToMarkdown/XmlMarkDown.cs

```diff
@@ -316,10 +316,30 @@ internal static string ToMarkDown(this XNode e)
                 string str = "";
```

Comment:
Hi @ramramps, we briefly updated this markdown generator so that it can output a warning message in the console whenever it comes across an empty `param` or `returns` tag. The warning message for now is just the method name so that we know where to look for that method and fix it. We could enhance the warning message, but for now it just does the job of highlighting the method name.